modules/cellranger/.gitignore

```diff
@@ -0,0 +1 @@
+cellranger-*.tar.gz
```

Comment:
Why is this file here?

Reply:
When you build the docker image you have to put cell ranger in the same directory. It's a just in-case anyone likes to `git add .` and `git commit`